taxonomy-editor-frontend/README.md

```diff
@@ -13,6 +13,9 @@ This is the main user interface developed in React, which works in conjunction w
 - [Material-UI](https://mui.com/)
 - [iso-639-1](https://www.npmjs.com/package/iso-639-1)
 
+This project was initially setup using [Create React App](https://github.com/facebook/create-react-app), [the documentation](https://create-react-app.dev/) might be useful.
+
+
```

Comment:
Thanks! Note that we should look to move to `Vite` soon, as `create-react-app` has been "deprecated" for quite some time (https://github.com/facebook/create-react-app/issues/13072) and library documentation tend to not support the CRA use case

Reply:
Ok, you know ;-)